packages/kbn-plugin-generator/sao_template/template/server/plugin.ts

```diff
@@ -0,0 +1,36 @@
+import { PluginInitializerContext, CoreSetup, CoreStart, Plugin } from '<%= relRoot %>/src/core/server';
+
+import { <%= upperCamelCaseName %>PluginSetup, <%= upperCamelCaseName %>PluginStart } from './types';
+
+export class <%= upperCamelCaseName %>ServerPlugin
+  implements Plugin<<%= upperCamelCaseName %>PluginSetup, <%= upperCamelCaseName %>PluginStart> {
+  constructor(initializerContext: PluginInitializerContext) {}
+
+  public setup(core: CoreSetup) {
+    const router = core.http.createRouter();
+
+    // Register a server side API endpoint
+    router.get(
+      {
+        path: '/api/<%= snakeCase(name) %>/example',
+        validate: false,
+      },
+      async (context, request, response) => {
+        return response.ok({
+          body: {
+            time: new Date().toISOString(),
+          },
+        });
+      }
+    );
+    return {};
```

Comment:
Following conventions, all routes should be defined in a `routes` subfolder. (see `x-pack/plugins/security/server/routes` or `src/core/server/capabilities/routes` for exemple)